### DISCUSSION TITLE
Template not up to date

### DISCUSSION THREAD

**Discussion by PenseeNumerique** [jasontaylordev/CleanArchitecture#608] [Q&A]
Hi,
If I'm not talking nonsense, the template via `dotnet new --install Clean.Architecture.Solution.Template` is not up to date.

I see [this commit ](https://github.com/jasontaylordev/CleanArchitecture/commit/35b490110f699c2ba427fd6b49e98babc16390c0), 
which updates Dockerfile in `src\WebUI` to version dotnet 6 and my file is still 5.

![image](https://user-images.githubusercontent.com/103261801/163563707-ab317bc8-c394-4e25-a73c-31e85a442afe.png)

Good day,

**Comment by bramburn** [comment]
that's docker, you can change the image to 6.0 but yes it seems out of date on that docker page. It's not a big deal.
